packages/langium-vscode/package.json

```diff
@@ -17,7 +17,7 @@
     "theme": "dark"
   },
   "engines": {
-    "vscode": "^1.67.0"
+    "vscode": "^1.84.0"
```

Comment:
I am not sure if this "target" is required to be as new is this, but on the other hand we now depend on vscode-languageclient/server 9.x. I don't know since when vscode requires it itself.

Reply:
Exactly that was my reason to upgrade `vscode`. And the VS Code IDE usually checks for updates itself so users probably won't have a (very) out-dated version,

Reply:
I agree. Does anyone else have a reason why we shouldn't?

Reply:
One word: Theia. If we don't benefit from any API changes (or need to update due to `vscode-languageclient`), we shouldn't update our required vscode version. This will only break compability with Theia users and accomplish nothing else.

Reply:
@msujew Actually, I have no clue what Theia does with this setting. Could you elaborate a bit?
Without any context the current settings looks like a magic number and appears just out-dated.